lib/presentation/utils/dashboard_pie_data.dart

```diff
@@ -0,0 +1,133 @@
+    ChartData('Clothes', 20, Colors.redAccent),
+    ChartData('Rents', 10, Colors.purpleAccent),
+    ChartData('Girls', 10, Colors.pink),
+    ChartData('Other', 10, Colors.orangeAccent),
+  ];
+  DateFormat format = DateFormat('MMMM');
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: double.infinity,
+      padding: const EdgeInsets.all(16),
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(16),
+        color: Colors.white,
+        border: Border.all(color: const Color(0xFFB8BBB4)),
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text(
+            "Parties",
+            style: Theme.of(context).textTheme.headlineSmall,
+          ),
+          Text(
+            LocaleKeys.fromDateToDate.tr(
+              args: [
+                format.format(DateTime.now()),
+                format.format(
+                  DateTime.now().add(
+                    const Duration(days: 30),
+                  ),
+                ),
+                DateTime.now().year.toString(),
+              ],
+            ),
+            style: Theme.of(context).textTheme.labelSmall,
+          ),
+          Center(
+            child: SizedBox(
+              width: 200.sp,
+              height: 200.sp,
+              child: SfCircularChart(
+                series: <CircularSeries>[
+                  PieSeries<ChartData, String>(
+                    dataSource: chartData,
+                    pointColorMapper: (ChartData data, _) => data.color,
+                    xValueMapper: (ChartData data, _) => data.property,
+                    yValueMapper: (ChartData data, _) => data.value,
+                    dataLabelSettings: const DataLabelSettings(
+                      isVisible: true,
+                      labelPosition: ChartDataLabelPosition.inside,
+                      labelIntersectAction: LabelIntersectAction.none,
+                      textStyle: TextStyle(
+                        fontSize: 12,
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                  )
+                ],
+              ),
+            ),
+          ),
+          GridView(
+            physics: const NeverScrollableScrollPhysics(),
+            shrinkWrap: true,
+            gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(
+              crossAxisCount: 2,
+              childAspectRatio: 4,
+            ),
+            children: chartData.map<Widget>((el) {
+              return Row(
+                spacing: 8,
+                children: [
+                  Icon(
+                    Icons.square,
+                    size: 16,
+                    color: el.color,
+                  ),
+                  Text(
+                    el.property,
+                    style: const TextStyle(
+                      fontSize: 16,
```

Comment:
The ScreenUtil class wraps the main app with the Flutter design size. It is advisable to continue using the extension getters, such as 'sp', 'h', 'w', etc., to adapt screen and font sizes to different screen sizes